jdaviz/core/template_mixin.py

```diff
@@ -2442,7 +2443,9 @@ def _update_mark_coords(self, *args):
                 mark.x, mark.y = x_coords, y_coords
 
     def get_mask(self, flux_cube, aperture_method,
-                 spectral_display_unit, reference_spectral_value=None):
+                 slice_display_unit, spatial_axes=(0, 1), reference_spectral_value=None):
+        # slice_axis is the remaining axis (of (0, 1, 2)) not included in spatial_axes
+        slice_axis = 3 - sum(spatial_axes)
```

Comment:
I was kind of mystified that this line works. I checked that it indeed works, but it feels like numerology? I've put in an equivalent but maybe less coincidental implementation below for your consideration.

```python
spatial_axes = [(0, 1), (1, 2), (0, 2)]
answers = [2, 0, 1]
for input, output in zip(spatial_axes, answers):

    assert 3 - sum(input) == output
    
    # suggested alternative:
    assert set([0, 1, 2]).difference(input).pop() == output
```

```suggestion
        slice_axis = set([0, 1, 2]).difference(input).pop()
```

Reply:
I can see how that's a little more obvious to read (although I do think the sum logic makes sense and isn't just a coincidence - it just only works when you want to find the single missing number from a set), but probably less performant?

Reply:
I've tested it, and of course it is less performant – but it's the difference between 40 nanoseconds for the original and 150 ns for my suggestion. I'm not sure anyone will notice. 😉